tools/importer-rest-api-specs/components/dataapigeneratorjson/models/object_definition.go

```diff
@@ -0,0 +1,12 @@
+package models
+
+type ObjectDefinition struct {
+	// Type specifies what kind of Object Definition this is (a Boolean, UserAssignedIdentityMap, etc).
+	Type ObjectDefinitionType `json:"Type"`
+
+	// ReferenceName specifies the name of the Constant or Model that this is a reference to (ie ConfigurationStoreProperties, EncryptionProperties)
+	ReferenceName *string `json:"ReferenceName,omitempty"`
+
+	// NestedItem is a nested ObjectDefinition when Type is a Dictionary or List
+	NestedItem *ObjectDefinition `json:"ObjectDefinition,omitempty"`
```

Comment:
the json name should match:

```suggestion
	NestedItem *ObjectDefinition `json:"NestedItem,omitempty"`
```